simple-todo-tdd/server/database/database.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+
+const MONGO_URL = 'mongodb://localhost:27017';
```

Comment:
Constants that may be different for each environment should be in an envoronment config file.